src/helpers/__test__/messageHelper.test.js

```diff
@@ -198,9 +216,18 @@ describe( 'messageHelper', () => {
                 };
                 expect( messageHelper.getMessageResponse( event ) ).toEqual( constants.USE_HERE_INSTEAD );
             } );
+
+            // On 6.2.2020, typing "@channel" in Slack gets sent to the bot as "!channel"
+            // we'll cover both cases in case Slack changes its mind.
+            test( '@converted to !', () => {
```

Comment:
Can you add a space between `@` and `converted`?

Reply:
Done.